php/libraries/Module.class.inc

```diff
@@ -138,8 +138,9 @@ class Module
         $identifier = $_REQUEST['identifier'] ?? '';
         $commentID  = $_REQUEST['commentID'] ?? '';
         $cls        = new $className($this, $page, $identifier, $commentID, $page);
-        // Smarty needs the Module name as a parameter to autoload the right
+        // Smarty needs the Module as a reference to autoload the right
         // templates. Every type of page uses this.
+        $cls->Module = $this;
```

Comment:
If this is the problem/fix, why is it only affecting the statistics module links?

I'd think nearly every page on LORIS should be broken.

Reply:
@driusan good question. I thought the same. I have an idea but not sure if it's right. I'll come up and talk to you about it at some point today